substrate/frame/grandpa/Cargo.toml

```
@@ -17,24 +17,22 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { features = ["derive"], workspace = true }
-frame-benchmarking = { optional = true, workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
 log = { workspace = true }
+scale-info = { features = ["derive", "serde"], workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

```suggestion
frame = { workspace = true, features = ["runtime"] }
```

I don't think we need this anymore.